src/tooltip.js

```diff
@@ -86,6 +86,10 @@ angular.module('cp-tooltip')
 					tooltipDisplayed = false;
 					if(tooltipEl) tooltipEl.remove();
 				}
+
+				function encodeHTML(str) {
+					return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
+				}
```

Comment:
http://stackoverflow.com/questions/295566/sanitize-rewrite-html-on-the-client-side